modules/ServerAPI/src/client/Protocol.css

```diff
@@ -0,0 +1,7 @@
+.select2, .select2-container, .select2-container--default {
```

Comment:
This reduces the width of the columns in the endpoint manager since it can become very wide due to the number of columns in the endpoint manager (especially when time/units & concentration/units are enabled)